dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -0,0 +1,112 @@
+[[getting-traffic-into-cluster]]
+= Getting Traffic Into The Cluster
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
+== Overview
+There are many ways to access the cluster. This section describes some
+commonly used approaches.
+
+The recommendation is:
+
+- When possible use the Router,
+- otherwise for TCP/UDP configure the service to provide access.
+
+The admin configures DNS records to point to IP addresses in the cluster
+that are exposed outside the cluster.  These addresses are commonly the
+IP addresses of nodes, Virtual IP addresses (VRRP) or Service NodePort
+addreses, Service externalIPs, Service ingressIP, etc..  The admin must
+make sure the routing tables permit the request to reach the IP address
+and iptables accept the packets as well.
```

Comment:
Can we say 'local firewall' rather than iptables.  Ideally they run firewalld now...